bot/cogs/help_channels.py

```diff
@@ -376,6 +376,12 @@ def is_dormant_message(message: t.Optional[discord.Message]) -> bool:
         embed = message.embeds[0]
         return embed.description.strip() == DORMANT_MSG.strip()
 
+    @staticmethod
+    def is_in_category(channel: discord.TextChannel, category_id: int) -> bool:
+        """Return True if `channel` is within a category with `category_id`."""
+        actual_category = getattr(channel, "category", None)
+        return actual_category and actual_category.id == category_id
```

Comment:
I feel like this will return `None` if the category isn't found.
```suggestion
        return actual_category is not None and actual_category.id == category_id
```

Reply:
Alternatively if this is intentional (since `None` will evaluate to `False` anyway in the conditionals) the return type could be annotated as optional. But I think it's better if it always returns a bool.

Reply:
Good catch